base/shell/cmd/lang/fr-FR.rc

```diff
@@ -374,7 +374,7 @@ START [""titre""] [/D chemin] [/I] [/B] [/MIN] [/MAX] [/WAIT]\n\
   B           Lance la commande ou le programme sans créer de fenêtre.\n\
   MIN         Démarre avec la fenêtre réduite.\n\
   MAX         Démarre avec la fenêtre agrandie.\n\
-  WAIT        Lance la commande ou le programme et attend qu'elle termine.\n\
+  WAIT        Lance la commande ou le programme et attend qu'elle se termine.\n\
```

Comment:
"qu'elle", or "qu'il" ? :D

Reply:
Good question. I have just added the missing word. I don't think that "iel" is allowed ;)

Reply:
yeah, if we could avoid the woke inclusive crap, it would be better.

Reply:
I have been desperately searching for the "règle d'accord avec des sujets coordonnés", for example here : https://www.projet-voltaire.fr/regles-orthographe/pierre-ou-paul-t-aidera-ou-pierre-ou-paul-t-aideront/. Between the rule of proximity and the rule of the masculine which prevails, it is not very clear. I think we can leave it as it is, Bernard Pivot will not come to pull our ears anymore.